src/pages/single-product.html

```diff
@@ -1,17 +1,387 @@
+                        </span>
+                    </div>
+                    <div class="flex flex-col gap-4 items-center">
+                        <img src="./../assets/images/icons/badge.png" alt="Support" class="h-[40px]">
+                        <span class="text-[#626E94] text-xs">
+                            قیمت مناسب
+                        </span>
+                    </div>
+                    <div class="flex flex-col gap-4 items-center">
+                        <img src="./../assets/images/icons/shopping-bag.png" alt="Support" class="h-[40px]">
+                        <span class="text-[#626E94] text-xs">
+                            ارسال به تمام کشور
+                        </span>
+                    </div>
+                    <div class="flex flex-col gap-4 items-center">
+                        <img src="./../assets/images/icons/safe-money.png" alt="Support" class="h-[40px]">
+                        <span class="text-[#626E94] text-xs">
+                            بازگشت وجه
+                        </span>
+                    </div>
+                </div>
+
+
+            </div>
+            <!-- ./short info -->
+
+            <!-- purchase box -->
+            <div class="flex flex-col justify-between basis-3/12 bg-[#F5F8FC] p-8">
+
+                <!-- top -->
+                <div class="flex flex-col">
+                    <h3 class="text-[20px] text-black font-bold">انتخاب رنگ</h3>
+                    <div class="flex flex-row gap-3 mt-2 text-[#9CB0B9]">
+                        <button class="flex flex-row gap-2 justify-center items-center bg-white border py-3 px-4">
+                            <span class="rounded-full bg-blue-600 w-[15px] h-[15px] inline-block"></span> <span>آبی</span>
+                        </button>
+                        <button class="flex flex-row gap-2 justify-center items-center bg-white border py-3 px-4">
+                            <span class="rounded-full bg-lime-500 w-[15px] h-[15px] inline-block"></span> <span>سبز</span>
+                        </button>
+                        <button class="flex flex-row gap-2 justify-center items-center bg-white border border-blue-400 text-black py-3 px-4">
+                            <span class="rounded-full bg-white border w-[15px] h-[15px] inline-block relative">
+                                <img src="../assets/images/icons/tick.png" alt="Tick icon" class="absolute left-[2px] bottom-[2px]">
+                            </span> <span>سفید</span>
+                        </button>
+                    </div>
+
+                    <hr class="bg-[#DEE2EEB2] my-[30px]">
+
+                    <h3 class="text-[20px] text-black font-bold">انتخاب سایز</h3>
+                    <div class="flex flex-row gap-2 mt-2 text-black">
+                        <button class="flex flex-row gap-2 justify-center items-center bg-white border py-3 px-4">
+                            <span>۱۰×۱۵</span>
+                        </button>
```

Comment:
The inner `<span>` element is unnecessary. You can just remove it and everything works like a charm.

```suggestion
                        <button class="flex flex-row gap-2 justify-center items-center bg-white border py-3 px-4">۱۰×۱۵</button>
```